### ISSUE TITLE
Check settings versus docs & schema

### ISSUE THREAD

**Issue by Jay-o-Way** [U-C-S/Hurl#163]
Compare https://raw.githubusercontent.com/U-C-S/Hurl/main/Utils/UserSettings.schema.json
What I found:

### Background type (enum) missing ~"none"~ "solid"

```json
        "BackgroundType": {
          "type": "string",
          "description": "The type of background to use for the browser window. mica or acrylic.",
          "enum": ["mica", "acrylic"]
```

### Rule prefixes: a prefix is expected, "no prefix" is not valid. However, according to docs, a prefix may be omitted to imply "string".

```json
          "Rules": {
            "type": "array",
            "items": {
              "type": "string",
              "description": "The URL pattern prefixed with s$, g$, r$ or d$, followed by the pattern to match. s$ = simple string, g$ = glob, r$ = regex, d$ = domain.",
              "pattern": "^(s\\$|g\\$|r\\$|d\\$).*"
            }
```

Rule Type | Format | Example
-- | -- | --
String | `s$__your_rule__` or `__your_rule__` | https://github.com/U-C-S

### Window minwidth and minheight

According to the docs, default size is 420×210. MainWindow properties are defined as:

```xaml
    MinWidth="460"
    MinHeight="230"
```

**Comment by Jay-o-Way** [comment]
Do we know of more inconsistencies?

**Comment by U-C-S** [comment]
Yeah, These 2 cases are the only inconsistencies as far as I know. I just left them as it is earlier before having the Settings UI to decide whether it should as it in the schema or as it in the code.

> s$__your_rule__ or __your_rule__

Both should be valid, so the schema's regex should be updated

> missing "none"

`none` currently doesn't work in the UI anyway, but want to support it in the future. we can leave the this as it is

**Comment by Jay-o-Way** [comment]
Let's collect the correct data here:

- Background type: Adding `"none"` to the enum is simple. Off-topic:
  - Maybe "solid" is a better word? _None_ can sound like _transparent_.
  - Should mica or acrylic be the default?
- Prefix: The best way to specify in RegEx that a prefix may be omitted is (...)
  - See also #166
- Minimum size: I think the code is leading, so **460 x 230**

**Comment by U-C-S** [comment]
`mica` is the default (acrylic has issues, i dont know if they still exist in newer versions of wpf-ui)
